server/bingo/tasks.py

```diff
@@ -0,0 +1,13 @@
+from .models import User
+from datetime import timedelta
+from django.utils import timezone
+
+
+def delete_inactive():
+    to_delete = User.objects.filter(is_active=False, created_at__lte=(
+        timezone.now() - timedelta(days=1)))
+    if count := to_delete.count():
+        to_delete.delete()
+        print(f"{count} inactive users deleted.")
```

Comment:
Calling `.count()` actually fetches all matching records, which is inefficient. Using `.exists()` first prevents an unnecessary database query.

So `.delete()` actually returns a tuple of (number of records deleted, details). Therefore instead of fetching .count() first, we can capture this return value.

Maybe you could write it something like

```
if to_delete.exists():
        deleted_count, _ = to_delete.delete()
```